### PR TITLE
adding 2 new js client demos

### PR DESCRIPTION
**Pull request by Cerchie**
adding a basic js client demo and a slightly more advanced version

### CODE REVIEW

File: confluent-kafka-javascript-poll/README.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  },
+  "question-7": {
+    "KStreams": 0,
+    "Flink": 0,
+    "lastClicked": false
+  }
+}
+```
+
+The code to seed your topic is set up for you in `seedproducer.js`. Run `node seedproducer.js` and check your topic in Confluent Cloud to confirm that the seed message arrived:
```
Comment (davetroiano) [inline confluent-kafka-javascript-poll/README.md:143]: need to add this file?

File: confluent-kafka-javascript-basic/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+`git clone https://github.com/Cerchie/basic-confluent-kafka-javascript-example.git` 
+
+`npm install` 
+
+## Step 2: Get set up in Confluent Cloud
+
+Sign up for [Confluent Cloud](https://www.confluent.io/confluent-cloud). 
+
+### To add an environment:
```
Comment (davetroiano) [inline confluent-kafka-javascript-basic/README.md:19]: optionally consider using the cloud quickstart CLI plugin to skip the env / cluster / credentials creation: not sure if it'd "just work" if you pass `--client nodejs` though

**Review by davetroiano — APPROVED** [review]
nicely done! one question and one comment to consider

**Review by sandonleejacobs — APPROVED** [review]
LGTM

**Review by sandonleejacobs — APPROVED** [review]

**Review by davetroiano — APPROVED** [review]
LGTM!

**Pull request merged**